html.go

```diff
@@ -748,7 +748,7 @@ func (r *HTMLRenderer) writeDocumentHeader(w *bytes.Buffer, sr *SPRenderer) {
 	w.WriteString(">\n")
 	if r.CSS != "" {
 		w.WriteString("  <link rel=\"stylesheet\" type=\"text/css\" href=\"")
-		attrEscape([]byte(r.CSS))
+		w.Write(attrEscape([]byte(r.CSS)))
```

Comment:
Yeah, all of that (lack of `Write`, `s/esc/attrEscape` and the unused parameter) are wrong and the result of hectic hacking. I'm pulling this in and will fix the rest right away.